src/content/docs/snowflake/features/api-integrations.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "API Integrations"
+description: Get started with API Integrations in LocalStack for Snowflake
+tags: ["Base"]
+---
+
+## Introduction
+
+API integrations in Snowflake provide a secure way to configure trust between Snowflake and external cloud providers such as AWS API Gateway. They are typically used when creating external functions or API-based workflows.
+
+The LocalStack Snowflake emulator now supports basic **CRUD operations** for API integrations. This allows you to `create`, `alter`, `drop`, `show`, and `describe` API integrations locally, which is useful for testing Terraform configurations or other automation flows that depend on these objects.
```

Comment:
same comment as in the other PRs with listing operations here and we only support CRUD operations for API Integrations, not the actual implementation or usage of them